packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json

```diff
@@ -2374,57 +2219,48 @@
     }
   ],
   "refresh": "5s",
-  "schemaVersion": 25,
+  "schemaVersion": 26,
   "style": "dark",
   "tags": [],
   "templating": {
     "list": [
       {
-        "allValue": null,
         "current": {
-          "selected": true,
-          "tags": [],
-          "text": "default",
-          "value": "default"
+          "selected": false,
+          "text": "v3",
+          "value": "v3"
```

Comment:
How about omitting any changes here and I will update with the new dynamic list selection after you merge it?

Reply:
now that CI passed, I'd love to move on -- feel free to improve this in your PR :)